.travis.yml

```diff
@@ -1,13 +1,49 @@
 language: java
 
-before_install:
-  - export MAVEN_OPTS=-Xmx1024m
-
-script:
-  - ./build/travis.sh
-
 jdk:
   - oraclejdk8
+  - oraclejdk9
```

Comment:
build don't work on Java9 or higher

Reply:
Oh sorry, this change wasn't meant to be in this PR. Just disable it.